src/hooks/usePantry.ts

```diff
@@ -194,6 +194,7 @@ async function getClosestPackageSuggestion(orgPkg: string): Promise<string> {
     pkgList.push(project)
   }
   for (const pkgName of pkgList) {
+    if(pkgName.includes(orgPkg)) return pkgName;
```

Comment:
seems a little overkeen, but we'll see how it goes. Approved and thanks.

Reply:
Your welcome :smile: